geode-web-management/src/main/java/org/apache/geode/management/internal/rest/ManagementLoggingFilter.java

```diff
@@ -31,13 +31,17 @@
 
 public class ManagementLoggingFilter extends OncePerRequestFilter {
 
+  // Because someone is going to want to disable this.
```

Comment:
remove this comment or give a description of what messages will be logged if it is set to true